packages/react/src/component-tests/IcPageHeader/IcPageHeaderTestData.tsx

```diff
@@ -0,0 +1,338 @@
+/* eslint-disable react/jsx-no-bind */
+
+import {
+  IcBreadcrumb,
+  IcBreadcrumbGroup,
+  IcButton,
+  IcChip,
+  IcNavigationItem,
+  IcStep,
+  IcStepper,
+  IcTextField,
+  IcTypography,
+} from "../..";
+/* eslint-disable react/jsx-no-bind */
+import React, { useRef, useState } from "react";
+import { IcPageHeader } from "../..";
+export const ScrollStickyPageHeader = () => {
+  return (
+    <>
+      <div style={{ margin: "10px" }}>
```

Comment:
Margin needs to be removed so that page header is full width
```suggestion
      <div>
```